config/templates/systemd/ovs-memcached.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=memcache
-After=local-fs.target network.target
-
-[Service]
-Type=simple
-ExecStart=/usr/bin/memcached -m 1000 -p 11211 -u memcached -l <MEMCACHE_NODE_IP>
```

Comment:
Is the default memcached listening on all ips? If not, we need to make sure it at least listens to the public ip selected in ```ovs setup```